env0/resource_approval_policy.go

```diff
@@ -0,0 +1,149 @@
+package env0
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const APPROVAL_POLICY = "approval-policy"
```

Comment:
I'd suggest to extract this type in the common template, and use the same one in every place (same for custom flow)

Reply:
sure!